### ISSUE TITLE
Any breaking change in version 0.3.1?

### ISSUE THREAD

**Issue by 5j9** [mediawiki-utilities/python-mwoauth#27]
pywikibot uses this library. After the new version release one of our oauth tests has started to fail.

The traceback I'm getting is this:
```
Traceback (most recent call last):
  File "...\Python36\lib\site-packages\mwoauth\functions.py", line 248, in identify
    resp = r.json()
  File "...\Python36\lib\site-packages\requests\models.py", line 858, in json
    self.content.decode(encoding), **kwargs
  File "...\Python36\lib\json\__init__.py", line 354, in loads
    return _default_decoder.decode(s)
  File "...\Python36\lib\json\decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "...\Python36\lib\json\decoder.py", line 357, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```

**Comment by minrk** [comment]
I've seen this as well - the response is not JSON itself, but jwt-encoded data.

**Comment by halfak** [comment]
Works for me.  

```
(3.5) [halfak@graphite: ~/projects/mwoauth]
$ python examples/handshaker.py 
Sending requests with default User-Agent.  Set 'user_agent' on mwoauth.flask.MWOAuth to quiet this message.
Point your browser to: https://en.wikipedia.org/w/index.php?title=Special%3AOAuth%2Fauthenticate&oauth_token=3de4f046872bed03db5892b44deae0f0&oauth_consumer_key=6decf90142f19c655a33dafbf92d2cdf
Response query string: oauth_verifier=4a92ec85990a331b7ac2882e10aefb64&oauth_token=3de4f046872bed03db5892b44deae0f0
AccessToken(key=b'6803bc740f6f686e1b0e2f585057a971', secret=b'5f715408e6b0326b6e2a194a3242c3e85135ee91')
Identified as EpochFail (id=6877667).
(3.5) [halfak@graphite: ~/projects/mwoauth]
$ python
Python 3.5.1+ (default, Mar 30 2016, 22:46:26) 
[GCC 5.3.1 20160330] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import mwoauth 
>>> mwoauth.__version__
'0.3.1'
```

**Comment by minrk** [comment]
Thanks, that's helpful. I'm not sure how it could work after #26 because `json.loads()` and `jwt.decode()` are called on the same text, which shouldn't ever work because jwt tokens aren't valid JSON and vice versa.

#28 makes the JSON parsing looking for errors conditional on starting with `{`, so it won't try to parse the access tokens.

**Comment by halfak** [comment]
Merged and uploaded 0.3.2 to pypi.
